pipeline/tasks/task-scripts/clean-up-old-appengine-versions.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+keepCount=${2:-"5"}
+
+function deleteOldVersions() {
+  _service=${1}
+  _keepCount=${2}
+
+  echo "Keeping last ${_keepCount} of app engine service ${_service}"
+  count=0
+  deletedCount=0
+  for version in `gcloud app versions list --service ${_service} --sort-by '~version.id' --format 'value(version.id)'`
+  do
+    ((count++))
+    if [[ ${count} -gt ${_keepCount} ]] ; then
```

Comment:
Depending on how smart we want our revision history to be (do we want to keep _any_ past 5 versions?), we could potentially extend this condition here with the functionality to keep a set of specific versions by comparing against a set of special release versions defined as k8s configMap key-value pairs like 
```
[...]
data = { 
  stable: "20190710t145511"
  unstable: "20190710t162254"
  testing: "20190710t163027"
}
```
As in: if `$version` matches "stable", skip deletion

Reply:
Could do, we can do that as a future iteration on this if it proves valuable. We'd need to have a mechanism for defining those states and when to flip them over.